### ISSUE TITLE
[SAFE] Adjust Review Claim screen

### ISSUE THREAD

**Issue by DmitryBespalov** [safe-global/safe-ios#2553]
- [x]  Design changes
    - [x]  “You’re claiming”, “Delegating to”
- [x]  MultiSend List
    - [x]  Update to the designs - for all multi-send details
    - [x]  Network banner
    - [x]  Change ellipsis algo: should be in the middle

https://www.figma.com/file/a0BQ5MEgKe6qsZXC5XTWHc/Claim-S-Tokens-Mobile-App?node-id=547%3A8370

**Comment by DmitryBespalov** [comment]
Note on MultiSend screens: for some reason, when opening the screen, the large title is not displayed automatically. I have spent some time on trying to make it work, but I found no way to enable it.

I leave it as is. If you scroll those screens just a little, the large title appears, otherwise it is a short title.

I have also not changed the title of the multi-send screen to "Contract Interaction", because it needs to be "MultiSend" since that is the name of the method call. The designs are incorrect in this.
